linera-client/src/persistent/file.rs

```diff
@@ -92,6 +93,13 @@ impl<T> std::ops::Deref for File<T> {
     }
 }
 
+impl<T> std::ops::DerefMut for File<T> {
```

Comment:
I'm not sure it really saves us anything here, since there aren't that many usages.  I think unless we use it in a lot of places and using a shorter import name saves some lines of code, I prefer to keep it self-contained so it's easier to refactor.